controllers_v2/keysend.ctrl.go

```diff
@@ -78,7 +78,7 @@ func (controller *KeySendController) KeySend(c echo.Context) error {
 	result, err := controller.SingleKeySend(c, &reqBody, userID)
 	if err != nil {
 		c.Logger().Errorf("Failed to send keysend: %s", err.Message)
-		return c.JSON(err.Code, err)
+		return c.JSON(http.StatusInternalServerError, err)
```

Comment:
when does this fail?

Reply:
See lndhub-notifications on slack, we're sending `2` as http status header

Reply:
yes, understood that. but why did the keysend fail? 

wondering if a 500 error is correct? 
and we probably should return a valid JSON response in the same format as others: https://github.com/getAlby/lndhub.go/blob/main/lib/responses/errors.go

Reply:
keysend fails because the user didn't have enough money.

Reply:
We are returning a valid JSON response. "err" is not an error (not the greatest var name here) it's a struct.

Reply:
if it is not enough money then it should be this error: https://github.com/getAlby/lndhub.go/blob/main/lib/responses/errors.go#L35

Reply:
if some server error then this: https://github.com/getAlby/lndhub.go/blob/main/lib/responses/errors.go#L17